simphony/io/tests/test_h5_lattice.py

```diff
@@ -73,9 +108,10 @@ def cleanup(self):
         shutil.rmtree(self.temp_dir)
 
     def container_factory(self, name, type_, base_vect, size, origin):
-        return H5Lattice.create_new(
-            self.handle.root, 'foo', type_, base_vect,
-            size, origin, record=CustomRecord)
+        self.group = self.handle.create_group(
```

Comment:
I would like to avoid using H5CUDS in order to keep the testing isolated and independent of the internals of H5CUDS. This mainly the goal of unit-testing vs integration testing.